index.js

```diff
@@ -209,6 +209,7 @@ const main = async () => {
   }
 
   const ignoredCounts = countIgnoredFindings(ignoredFindings)
+  const findings_details = findings.imageScanFindings.findings || []
```

Comment:
nit pick (consistency): `findingsDetails` camelCase instead of snake_case

```suggestion
  const findingsDetails = findings.imageScanFindings.findings || []
```